frontend/src/modules/tag/components/tag-popover.vue

```diff
@@ -1,107 +1,111 @@
 <template>
-  <app-teleport to="#teleport-modal">
-    <app-dialog
-      v-model="computedVisible"
-      title="Edit tags"
-      :pre-title="pretitle"
-    >
-      <template #content>
-        <div class="px-6 pb-6">
-          <form v-if="visible" class="tags-form">
-            <app-tag-autocomplete-input
-              v-model="model"
-              :fetch-fn="fields.tags.fetchFn"
-              :mapper-fn="fields.tags.mapperFn"
-              :create-if-not-found="true"
-              placeholder="Type to search/create tags"
-            />
-          </form>
-        </div>
+  <app-dialog
+    v-if="computedVisible"
+    v-model="computedVisible"
+    title="Edit tags"
+    :pre-title="modelValue.displayName"
```

Comment:
I think you should still pass "preTitle" through props. Currently, this component is being used in "member-list-bulk-update-tags and "tags-list". I don't think "modelValue.displayName" can be applied in both